### DISCUSSION TITLE
Breaking news: Himalaya receives financial support!

### DISCUSSION THREAD

**Discussion by soywod** [pimalaya/himalaya#399] [Show and tell]
I'm proud to announce that the Himalaya project will receive financial support from the [NLnet foundation](https://nlnet.nl/project/Himalaya/) :tada:
This support will allow me to dedicate a good half of my time to the project till we reach the first production-ready version. I will publish soon the full planning. Part of the donation is available for anyone who wants to contribute, please reach me [by email](mailto:clement.douin@posteo.net) if you are interested in.

Second big news: I planned to move away from GitHub. I don't like the path they are taking, especially after what happened with Copilot. I planned to move everything to [sourcehut](https://git.sr.ht/~soywod/), it will be the good occasion to split the project into small repositories (one for the lib, and one per client like CLI, TUI, GUI, plugins etc). I will let you know whenever the transfert will be effective.

**Comment by hao-lee** [comment]
Glad to hear that. I have been looking forward to the production version.

**Comment by kskarthik** [comment]
I truly appreciate your decision on moving to sourcehut :+1:

**Comment by soywod** [comment]
The financial support is set up, it will start from the 12th of September. Starting from this date, the project should be really active :tada:

**Reply by kskarthik** [reply]
Glad to know! Are you planning for the move to soucrehut too, in the timeframe?

**Reply by soywod** [reply]
Yes, it will be the first task in the pipeline. I was thinking to move everything expect for the CLI, because GitHub still gives good visibility for the project. Plus Himalaya was just a CLI before turning into a lib. I find it a good deal to keep the original CLI repo on GitHub (not to confuse existing users + for the visibility) and to move all the rest on Sourcehut (lib, GUI, plugins etc).

**Reply by choucavalier** [reply]
Really cool! Can't wait to use himalaya once it's more mature :)

**Reply by holms** [reply]
What's wrong with copilot?

**Reply by soywod** [reply]
I personally find nothing wrong about the project itself, what bothers me is the way it is used. I advise you to read [this post](https://sfconservancy.org/blog/2022/jun/30/give-up-github-launch/) to have some more context.
